Group5 - UserStories

```diff
@@ -22,6 +22,7 @@ US007 - As a user, I want to save PDFs as an image files such as PNG or JPEG to
 Sanyam Goyal
 ~~US009 - As a user, I want to have a feature to extract text and images from different types of file formats, so I can use different types of docuements.~~
 US010 - As a user, I want to have a feature to delete a page from the pdf document. 
+US019 - As a user, I want to have a feature to add images to a PDF document.
```

Comment:
This would be a nice feature to have for our project. I will go ahead and merge this pull request.